catalyst/callbacks/scheduler.py

```diff
@@ -116,55 +119,89 @@ def _scheduler_step(
     ):
         if isinstance(scheduler, torch.optim.lr_scheduler.ReduceLROnPlateau):
             scheduler.step(reduced_metric)
-            lr = scheduler.optimizer.param_groups[0]["lr"]
         else:
             scheduler.step()
-            lr = scheduler.get_lr()[0]
 
-        momentum = get_optimizer_momentum(scheduler.optimizer)
+        lr_list = [
+            param_group["lr"]
+            for param_group in scheduler.optimizer.param_groups
+        ]
+        momentum_list = get_optimizer_momentum_list(scheduler.optimizer)
+        return lr_list, momentum_list
 
-        return lr, momentum
-
-    def step_batch(self, runner: "IRunner") -> None:
-        """Update learning rate and momentum in runner.
+    def _update_lr_and_momentum_in_metrics_dict(
+        self, metrics_dict, lr_list, momentum_list
+    ):
+        """Update learning rate and momentum in metrics_dict
 
         Args:
-            runner: current runner
+            metrics_dict (dict): batch_metrics or epoch_metrics
+            lr_list (List[float]): lr for each param group
+            momentum_list (List[float, None]): momentum for each param group
```

Comment:
but remove them (typings) from the docs?

Reply:
done